kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbLocalQueryService.java

```diff
@@ -16,12 +19,17 @@ public KaldbLocalQueryService(ChunkManager<T> chunkManager) {
 
   @Override
   public KaldbSearch.SearchResult doSearch(KaldbSearch.SearchRequest request) {
+    LOG.info("Received search request: {}", request);
     ScopedSpan span = Tracing.currentTracer().startScopedSpan("KaldbLocalQueryService.doSearch");
     SearchQuery query = SearchResultUtils.fromSearchRequest(request);
+    span.tag("query", query.toString());
     SearchResult<T> searchResult;
     searchResult = chunkManager.query(query);
     KaldbSearch.SearchResult result = SearchResultUtils.toSearchResultProto(searchResult);
+    span.tag("total_nodes", String.valueOf(result.getTotalNodes()));
```

Comment:
Same as above, both of these should be camel cased

Reply:
Good point. Fixed.